### PR TITLE
towire uuid

### PR DESCRIPTION
**Pull request by ragibkl**
What it says on the tin.

### CODE REVIEW

File: convergence/src/to_wire.rs [hunk 1]
```diff
@@ -101,6 +101,17 @@ impl ToWire for Decimal {
 	}
 }
 
+impl ToWire for uuid::Uuid {
+	fn to_binary(&self) -> Vec<u8> {
+		let mut b = BytesMut::new();
+		self.to_sql(&postgres_types::Type::UUID, &mut b).unwrap();
```
Comment (tobyhede) [inline convergence/src/to_wire.rs]: The actual implementation of to_sql is just a call to `self.as_bytes()` 
Function could be `self.as_bytes().into()`

**Review by tobyhede — APPROVED** [review]
Might be worth adding a test for the encoding.

File: convergence/src/to_wire.rs (hunk 1 above)
Comment (ragibkl) [reply convergence/src/to_wire.rs]: Nice one! Updated.

**Comment by ragibkl** [comment]
> Might be worth adding a test for the encoding.

Added. Looks like I added a bunch of towire impl without the tests. I'll do those separately.

**Pull request merged**